src/main/java/com/minecolonies/coremod/proxy/IProxy.java

```diff
@@ -2,6 +2,9 @@
 
 import com.minecolonies.coremod.client.gui.WindowBuildTool;
 import com.minecolonies.coremod.colony.CitizenDataView;
+import net.minecraft.client.util.RecipeBookClient;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'net.minecraft.client.util.RecipeBookClient'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)